dev/arktype.io/docs/api/validatebound.md

```diff
@@ -0,0 +1,23 @@
+---
+hide_table_of_contents: true
+---
+
+# validateBound
+
+## operator
+
+-   [bound](./validatebound.md)
+
+## string
+
+-   "[value?][comparitor?]type[comparitor][value]"
```

Comment:
I'd split this up like:

```md
N: number literal
S: Sized data (a number, string or array)
<: Comparator (one of  "<", "<=", "==", ">=", ">")

### Bound
"S<N"

### Range

"N<S<N", with comparators restricted to `<` or `<=`
```

I'm sure the formatting on the markdown sucks, especially for the key at the top but maybe you know a better way to do that